contracts/remote-address-validator/RemoteAddressValidator.sol

```diff
@@ -16,8 +16,12 @@ contract RemoteAddressValidator is IRemoteAddressValidator, Upgradable {
     mapping(string => string) public remoteAddresses;
     address public immutable interchainTokenServiceAddress;
     bytes32 public immutable interchainTokenServiceAddressHash;
```

Comment:
this is ununsed now?

Reply:
fine to keep, but maybe add a getter for it then, so it's easily queryable?

Reply:
it is public so it is easily queryable. (interchainTokenServiceAddress() is a function that can be called)